dvc/objects/__init__.py

```diff
@@ -1,14 +1,31 @@
 import logging
 from concurrent.futures import ThreadPoolExecutor, as_completed
+from typing import TYPE_CHECKING, NamedTuple, Optional, Set
 
 from dvc.progress import Tqdm
 
 from .tree import Tree
 
+if TYPE_CHECKING:
+    from dvc.remote.base import Remote
+
+    from .db.base import ObjectDB
+    from .file import HashFile
+
 logger = logging.getLogger(__name__)
 
 
-def save(odb, obj, jobs=None, **kwargs):
+class UsedObjectsPair(NamedTuple):
+    remote: Optional["Remote"]
+    objs: Set["HashFile"]
```

Comment:
This whole thing is temporary, I think what we want (for now) is to return a dict mapping odbs to a set of objects